cScripts/functions/vehicle/fn_vehicle_addInventory.sqf

```diff
@@ -231,91 +233,14 @@ if (_vehicle iskindOf "rhsusf_stryker_base") then {
     [_vehicle, 6, 38, false, false] call FUNC(setCargoAttributes);
 
     switch (_vehicleType) do {
+        case "rhsusf_stryker_m1126_m2_d";
+        case "rhsusf_stryker_m1126_m2_wd";
```

Comment:
Dont think its greate to declare them all as medical vehicles. Better using the variable system already implemented.

Reply:
I'm going to re-add the MED case but will keep these as default medical vehicles per my COC. Same with Soft Top 2 Door. Will leave the MED case so that other strykers and humvees can be made medical vehicles if so desired.

Reply:
Additionally I will update the MED changes for the radio. Strykers do not seem to be added yet.